docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,23 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned as a row for each table's column, plus a summary row for
+each table. The following table lists the returned columns and what statistics.
+they represent. Any additional statistics collected on the data source, other
+than those listed here, are not included.
 
-==========================  =============================================================
-Column                      Description
-==========================  =============================================================
-``column_name``             The name of the column (``NULL`` for the summary row)
-``data_size``               The total size in bytes of all of the values in the column
-``distinct_values_count``   The number of distinct values in the column
+==========================  ============================================================= =================================
+Column                      Description                                                   Notes
+==========================  ============================================================= =================================
+``column_name``             The name of the column                                        Returned in column rows
+``data_size``               The total size in bytes of all of the values in the column    Only counts string type values (``CHAR``, ``VARCHAR``, etc)
+``distinct_values_count``   The estimated number of distinct values in the column
 ``nulls_fractions``         The portion of the values in the column that are ``NULL``
-``row_count``               The number of rows (only returned for the summary row)
-``low_value``               The lowest value found in this column (only for some types)
-``high_value``              The highest value found in this column (only for some types)
-==========================  =============================================================
+``row_count``               The estimated number of rows in the table                     Returned in table summary rows
+``low_value``               The lowest value found in this column                         Only counts numeric type values (``BIGINT``, ``DECIMAL``, etc)
+``high_value``              The highest value found in this column                        Only counts numeric type values (``BIGINT``, ``DECIMAL``, etc)
```

Comment:
Slight correction. Instead of using "counts" in low_value, high_value and data_size rows maybe use something like:

`Available for columns of textual types (...)` or `Available for columns of numeric types (...)`

"counts" conveys there can be non-numeric values too within a numeric column.